packages/expo/src/launch/SplashScreen.ts

```diff
@@ -1,11 +1,17 @@
-import { NativeModules } from 'react-native';
+import { NativeModules, Platform } from 'react-native';
 
-const { ExponentSplashScreen: SplashScreen } = NativeModules;
+const { ExponentSplashScreen: SplashScreen = {} } = NativeModules;
+
+const isWeb = Platform.OS === 'web';
 
 export function preventAutoHide() {
-  SplashScreen.preventAutoHide();
+  if (!isWeb) {
```

Comment:
Instead of making assumptions about the platform, feature detection is simple here and more scalable when adding more platforms:

```suggestion
  if (SplashScreen.preventAutoHide) {
```